packages/fxa-settings/src/pages/ResetPassword/index.tsx

```diff
@@ -0,0 +1,190 @@
+      entrypoint_variation: 'react',
+    });
+    try {
+      // TODO resetPassword function in FXA-6123
+      account.resetPassword(email);
+      navigateToConfirmPwReset();
+    } catch (e) {
+      const errorResetPassword = ftlMsgResolver.getMsg(
+        'reset-password-error-general',
+        'Sorry, there was a problem resetting your password'
+      );
+      alertBar.error(errorResetPassword);
+    }
+  };
+
+  return (
+    <>
+      <CardHeader
+        headingWithDefaultServiceFtlId="reset-password-heading-w-default-service"
+        headingWithCustomServiceFtlId="reset-password-heading-w-custom-service"
+        headingText="Reset password"
+        {...{ serviceName }}
+      />
+
+      <WarningMessage
+        warningMessageFtlId="reset-password-warning-message"
+        warningType="Note"
+        warningMessage="When you reset your password, you reset your account. You may lose some of your personal information (including history, bookmarks, and passwords). That’s because we encrypt your data with your password to protect your privacy. You’ll still keep any subscriptions you may have and Pocket data will not be affected."
+      />
+
+      <form
+        noValidate
+        className="flex flex-col gap-4"
+        onSubmit={handleSubmit(onSubmit)}
+        data-testid="reset-password-form"
+      >
+        {/* if email is forced, display a read-only email */}
+        {/* do not provide input field to modify the email */}
+        {forceEmail && (
+          <p
+            data-testid="reset-password-force-email"
+            className="text-base break-all"
+          >
+            {forceEmail}
+          </p>
+        )}
+
+        {/* if email is not forced, display input field */}
+        {!forceEmail && (
+          <InputText
+            type="email"
+            label="Email"
+            name="email"
+            placeholder={'username@domain.com'}
+            onChange={(e) => {
+              setEmail(e.target.value);
+              // clear error tooltip if user types in the field
+              if (emailErrorText) {
+                setEmailErrorText('');
+              }
+            }}
+            onFocusCb={onFocusMetricsEvent ? onFocus : undefined}
+            autoFocus
+            errorText={emailErrorText}
+            className="text-start"
+            anchorStart
+            autoComplete="off"
+            spellCheck={false}
+            prefixDataTestId="reset-password"
+          />
+        )}
+
+        <FtlMsg id="reset-password-button">
+          <button
+            data-testid="reset-password-button"
+            type="submit"
+            className="cta-primary cta-base-p leading-none py-4"
+          >
+            Begin Reset
+          </button>
+        </FtlMsg>
+      </form>
+
+      {canGoBack && !forceEmail && <LinkRememberPassword email={email} />}
+      {canGoBack && forceEmail && (
+        <LinkRememberPassword forceEmail={forceEmail} />
```

Comment:
Could have some syntactic sugar here like `<LinkRememberPassword {...{forceEmail }} />` and on the other line but 🤷‍♀️

Reply:
Oh yesss thank you